src/applications/disability-benefits/all-claims/components/ReviewCardField.jsx

```diff
@@ -142,14 +174,23 @@ export default class ReviewCardField extends React.Component {
             className="usa-button-primary update-button"
             onClick={this.update}
           >
-            Done
+            {volatileData ? 'Save' : 'Done'}
```

Comment:
I was thinking this might be better the other way around, because of the interaction you cited in the PR description wherein the data is updated without a user needing to actually hit 'save'. If that's the case, 'save' might be misleading since the data's already saved, and 'done' might be a better fit. Thoughts?

Reply:
Yeah, I mean in either case (`volatileData` or no), hitting the button is more of a "done."

@goldenmeanie What do you think? (Context for this can be found in the original PR comment up top.)